src/server/webSocket/socketManager.js

```diff
@@ -1,20 +1,35 @@
 import socketio from 'socket.io';
 
+import Game from '../classes/Game';
+import Player from '../classes/Player';
 import logger from '../helpers/logger';
 
 let io = null;
 
+let allGames = [new Game({ room: 'oijf9898a' }), new Game({ room: 'ffhreuf8fhf' })];
+
 /**
  * handle game socket input
  *
  * @param {Object} data - the data
  * @return {void}
  */
 export async function handleGameSocket(data) {
-  if (data.path === '/new') {
-    // do new game
-
-    // io.sockets.to().emit('', );
+  const { path } = data;
+  switch (path) {
+    case '/join': {
+      console.log('new peer joined the game', data);
+      const { room, id, socket } = data;
```

Comment:
here have we check if variables exist ? and handle fallback if not, or there is no reason to have event without good params ?

Reply:
if var don't exist we will get undefined, maybe we should just check that in our code.
it would be better/easier to handle bad input this way (test undefined) than implement a checking function imo